src/Test/AppModel/DRT/Application/AppShutdown/DrtAppShutdown.cs

```diff
@@ -0,0 +1,78 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace DRT
+{
+/// <summary>
+/// Calling Shutdown from inside a dialog causes InvalidOperationException
+/// The problem is actually caused by calling Shutdown with more than one window is open.
+/// 
+/// Note that this DRT does not assert anything, its failure mode is an exception on shutdown.
+/// </summary>
+    class DrtAppShutdown : Application
+    {
+        private Logger _logger = new Logger("DrtAppShutdown", "Microsoft", "Tests app shutdown when multiple windows open that have owned windows");
+        const int WindowCount = 2; // We just need more than one.
+
+        static bool passed = false;
```

Comment:
Code format issues.

Reply:
@lindexi , I ran the code formatter, but it did not make any changes. What is the issue you are mentioning in this comment ??

Reply:
@dipeshmsft Sorry. Please see https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md

> We use `_camelCase` for internal and private fields and use readonly where possible. Prefix internal and private instance fields with `_` , static fields with s_ and thread static fields with t_. When used on static fields, readonly should come after static (e.g. static readonly not readonly static). Public fields should be used sparingly and should use PascalCasing with no prefix when used.